### PR TITLE
Setup pop up window

### PR DESCRIPTION
**Pull request by JhonatanErosa**
### PopUp Window

_________________________

- Create Projects Array in JavaScript
- Add Functions createCard and showPopUp
- Add CSS styles for mobile and desktop versions
- Fix linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,8 +55,8 @@ <h2>I'm a Software Developer</h2>
             <h2>My Recent Works</h2>
             <div class="topLine"></div>
         </div>
-        <div class="grid">
-            <div class="mainCard">
+        <div class="grid" id="projectsSection">
+            <!-- <div class="mainCard">
                 <div class="cardImage">
                     <img src="Sources/MultiPost Stories.svg" alt="MultiPostImage">
                 </div>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] kindly remove the commented piece of code for cleaner code practices. This is to follow the YAGNI  principle, which implies that you should only implement needed material in your code.

File: script.js [hunk 2]
```diff
@@ -15,4 +15,179 @@ hamburger.addEventListener('click', openWindow);
 closeIcon.addEventListener('click', closeWindow);
 for (let i = 0; i < linksPopUp.length; i += 1) {
   linksPopUp[i].addEventListener('click', closeWindow);
+}
+
+// PopUp Window
+const projects = [
+  {
+    name: 'Multi-Post Stories',
```
Comment (tufoinnkuo10) [inline script.js:23]: - [ ]please kindly reduce the gaps between the tech buttons on the top left of the desktop popup, and that of the mobile version, so it matches the original design.
yours
Screenshot (1277)
Screenshot (1279)
figma
Screenshot (1278)
Screenshot (1280)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Team

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ popup well responsive

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by DeliceLydia — APPROVED** [review]
## Hi @JhonatanErosa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Design looking nice



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**